templates/Canvas/MVVM Module.xctemplate/___FILEBASENAME___/ViewModel/___VARIABLE_moduleName___ViewModel.swift

```diff
@@ -0,0 +1,16 @@
+//___FILEHEADER___
+
+import Combine
+
+public class ___VARIABLE_moduleName___ViewModel: ObservableObject {
+    @Published public private(set) var state: InstUI.ScreenState = .loading
+    public let pageTitle = String(localized: "template")
```

Comment:
Maybe we could use  `""` here to avoid unnecessary Localizable changes.

Reply:
I like that the "template" string makes sure I don't forget to update it.

Reply:
In light of recent findings, please add `, bundle: .core`
I also found the text `"DEVELOPER"` already available in translations, it could be a good placeholder candidate to jump at you in all caps if you forget :)